### DISCUSSION TITLE
Item names/offsets?

### DISCUSSION THREAD

**Discussion by o777l** [winstxnhdw/lc-hax#394] [General]
Anyone have a list of item names or offsets for them? Or how to get those?

Trying to /grab specific items in game, 'mask', 'comedy' and 'tragedy' won't work etc. even when spawned

**Comment by winstxnhdw** [comment]
The names are whatever you see on ESP. Which also means that `comedy`, `tragedy` and `mask` will work. The mask could be bugged. Try `/mask` which will guarantee to pick up a mask to know.

**Comment by winstxnhdw** [comment]
Also the search is via fuzzy finding. You shouldn’t have to write their exact names to grab them. For example, the full name for mask is `Comedymask` but `comedy` or `mask` or even `coedymaks` should retrieve them correctly.
